### PR TITLE
Support setting ingress/egress priority maps

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

**Review by markgoddard — COMMENTED** [review]
Would these settings make sense on a bond or bridge interface?

File: templates/ethernet_nmconnection.j2 [hunk 1]
```diff
@@ -25,6 +25,12 @@ mtu={{ item.mtu }}
 interface-name={{ item.device }}
 parent={{ item.device | regex_replace(vlan_interface_regex, '\g<interface>') }}
 id={{ item.device | regex_replace(vlan_interface_regex, '\g<vlan_id>') }}
+{% if item.ingress_priority_map is defined %}
+ingress-priority-map={{ item.ingress_priority_map }}
```
Comment (markgoddard) [inline templates/ethernet_nmconnection.j2:29]: It looks like there is an equivalent setting in network-scripts: https://people.freedesktop.org/~lkundrak/nm-docs/nm-settings-ifcfg-rh.html

**Comment by MoteHue** [comment]
> Would these settings make sense on a bond or bridge interface?

Yes I believe so, afaik these can be applied in the same situations a VLAN id can. I've been testing with both bridges and bonded interfaces and haven't seen issues with applying the options.

**Comment by oneswig** [comment]
> Would these settings make sense on a bond or bridge interface?

No - it has to be a VLAN interface.  The QOS bits are 3 bits at the top of a VLAN tag.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**